### PR TITLE
Fix examples

### PR DESCRIPTION
**Pull request by malfter**
##### SUMMARY

Fix module example

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

dns record

### CODE REVIEW

File: plugins/modules/hetzner_dns_record.py [hunk 1]
```diff
@@ -54,7 +54,7 @@
 
 - name: Remove a new.foo.com A record
   community.dns.hosttech_dns_record_set:
```
Comment (markuman) [inline plugins/modules/hetzner_dns_record.py:56]: The used module is also wrong here.

Otherwise LGTM

Comment (malfter) [reply plugins/modules/hetzner_dns_record.py]: I have also corrected the module.

File: plugins/modules/hetzner_dns_record.py [hunk 2]
```diff
@@ -53,8 +53,8 @@
     hetzner_token: access_token
 
 - name: Remove a new.foo.com A record
-  community.dns.hosttech_dns_record_set:
-    state: present
+  community.dns.hosttech_dns_record:
```
Comment (felixfontein) [inline plugins/modules/hetzner_dns_record.py:56]:
```suggestion
  community.dns.hetzner_dns_record:
```
This is an example for hetzner_dns_record (see the module's filename).

Comment (malfter) [reply plugins/modules/hetzner_dns_record.py]: You're right, that was a bit quick of me to adjust.

**Review by markuman — APPROVED** [review]

**Comment by felixfontein** [comment]
The same error happens in plugins/modules/hosttech_dns_record.py (both module name and `state: present` instead of `state: absent`). Would you mind fixing that one as well in this PR?

**Comment by malfter** [comment]
Yes, I can do that with pleasure. Is it okay for you if I combine all changes into one commit?

**Comment by felixfontein** [comment]
Sure, that's fine! (We also squash on merge, so having more than one commit doesn't matter.)

**Review by markuman — APPROVED** [review]

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@malfter thanks a lot for fixing this!
@markuman thanks for reviewing!

**Pull request merged**